demo/installer/armv8a/install.sh

```diff
@@ -0,0 +1,422 @@
+#!/bin/sh
```

Comment:
This whole file looks very similar to the x86_64 file.

If the installer files are very similar between x86_64 and aarch64 then I propose doing what we did for powerpc and arm32 (which are similar).  In that case we called them "u-boot-arch".

In this case perhaps we can go with "grub-arch" as both x86_64 and aarch64 appear to be GRUB based.